src/main/java/org/takes/facets/fork/FkHitRefresh.java

```diff
@@ -248,7 +243,7 @@ private boolean directoryUpdated() {
             }
             final File[] files = this.dir.listFiles();
             boolean expired = false;
-            if (files != null) {
+            if (new Opt.Single<>(files).has()) {
```

Comment:
@Iprogrammerr I'm sorry (and I don't want to be rude) but I think you did very few work here. You just removed one null, changed a uncessary code and added a lot of todo (PDD puzzle) comments. My suggestion to you is remove more nulls.